src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml

```diff
@@ -14,6 +14,11 @@
     <link simpl-append-version="true" rel="stylesheet" href="~/lib/multi-dropdown-menu/css/bootstrap-4-navbar.css" />
     <link simpl-append-version="true" rel="stylesheet" href="~/modules/core/site.css" />
 
+    @{
+        <link simpl-append-version="true" rel="stylesheet" href="~/lib/smoothproducts/css/smoothproducts.css" />
+        <link simpl-append-version="true" rel="stylesheet" href="~/modules/catalog/product-detail.css" />
+    }
+
```

Comment:
why need to put in this @{ } those file will need in other Layout also ?

Reply:
I added the script files to layout for now, just for now,  will need find another way to handle this. 
Partial view is not able to refer to any css files for some reason.There is one similar issue for your reference "https://stackoverflow.com/questions/5110028/add-css-or-javascript-files-to-layout-head-from-views-or-partial-views".

Reply:
done